src/excmds.ts

```diff
@@ -427,10 +427,11 @@ export async function clipboard(excmd = "open", content = ""){
 
 // {{{ Buffer/completion stuff
 // TODO: Move autocompletions out of excmds.
-
-/** @hidden */
-//#background_helper
-const DEFAULT_FAVICON = browser.extension.getURL("static/defaultFavicon.svg")
+/** Ported from Vimperator. */
+//#background
+export async function tabs() { fillcmdline("buffer") }
+//#background
+export async function buffers() { tabs() }
```

Comment:
macros.py can't handle function bodies on the same line as the function signature.